src/plugins/console/public/lib/kb/kb.test.js

```diff
@@ -133,12 +133,12 @@ describe('Knowledge base', () => {
   );
 
   indexTest('Index integration 2', ['index1'], [], {
-    index: ['index1'],
```

Comment:
changes in this test files is the reverse of changes in this PR https://github.com/elastic/kibana/pull/162917